apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js

```diff
@@ -80,6 +87,100 @@ describe('ExportPaymentsService', () => {
         })
     })
 
+    it('check payments export', async () => {
+        const locale = 'ru'
+        const { admin, billingReceipts, acquiringContext, acquiringIntegration, organization } = await makePayer()
```

Comment:
best to export admin and regular data to beforeAll()

Reply:
I agree, but this utility is used in many places, it’s not something new. Most likely, need to create an admin in `beforAll` and refactor it in the tests that use this utility. I'm not ready to do it in this pull request

Reply:
On the contrary, this functionality is rather new, and we have agreed that we are going to use beforeAll long before this functionality

Reply:
So I really think that it is best for this PR to implement beforeAll